File: java/src/model/chimpos/Zapato.java
package model.chimpos;

import model.ataque.Ataque;
import model.ataque.Zapatazo;
import model.tipo.TipoChinpokomon;

import java.util.ArrayList;
import java.util.Arrays;

public class Zapato extends Chinpokomon {

    public Zapato(String nombre, Integer nivel, TipoChinpokomon tipo) {
        super(nombre, nivel, 30, new ArrayList<Ataque>(Arrays.asList(new Zapatazo())), tipo);
    }

}

File: python/model/chipos/Zapato.py
from ataque.Zapatazo import Zapatazo
from chipos.Chipokomon import Chipokomon


class Zapato(Chipokomon):
    def __init__(self, nombre, nivel, tipo):
        Chipokomon.__init__(self,nombre, nivel, 30,[Zapatazo], tipo)

    def __str__(self):
        return 'Zapato'